### ISSUE TITLE
[Incompatibility] DNDBeyond Importer Button duplicates twice when adding an Item

### ISSUE THREAD

**Issue by Gensokian** [kgar/foundry-vtt-tidy-5e-sheets#428]
This is a weird one and might be a problem with DDB Importer (Feel free to close this if so)
Every time I add any item to my character sheet, it adds 2 buttons for the DDB importer! But only ONCE!
Reopening the sheet fixes the problem

I have no clue why; but it happens :)
https://github.com/kgar/foundry-vtt-tidy-5e-sheets/assets/9213561/a5e8964a-a633-4966-afcd-f74b2fac206d

**Comment by SirAron111** [comment]
Happens to me to could maybe be related to another issue that causes double renders

**Comment by kgar** [comment]
What version of Tidy are you using?

**Comment by kgar** [comment]
I did a release a day or two ago to address the double (triple?) rendering problem.
And I just now pushed another release for unidentified item support.

**Comment by SirAron111** [comment]
Seems fixed for me n the newer versions. Doesn't double render anything anymore.

**Comment by Gensokian** [comment]
Sorry for the long delay!
I did just update from 0.2.20 (Was latest when I checked it) ⇾ 2.4.1
This seems to work fine now!
